internal/schema/stacks/1.8/required_provider_block.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/hcl-lang/lang"
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/hashicorp/terraform-schema/internal/schema/refscope"
+	"github.com/hashicorp/terraform-schema/internal/schema/tokmod"
+)
+
+func requiredProviderBlockSchema() *schema.BlockSchema {
```

Comment:
Was this copy pasted and not adjusted yet? I think required providers needs a different structure 😁